src/main/java/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package service.impl;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+import service.FileReaderService;
+
+public class FileReaderServiceImpl implements FileReaderService {
+
+    @Override
+    public List<FruitTransaction> readFromFile(File activitiesFile) {

```

Comment:
do not parse transactions while reading from file

Reply:
Done